include/revng/Model/LoadModelPass.h

```diff
@@ -43,17 +45,27 @@ class LoadModelWrapperPass : public llvm::ImmutablePass {
   static char ID;
 
 private:
+  ModelWrapper *ExternalModel;
   std::optional<ModelWrapper> Wrapper;
```

Comment:
`std::variant`?

Reply:
i can't :( they have the same type and one is owning and the other one not

Reply:
Same type, but one is a pointer, the other is a regular instance.